src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseFileService.cs

```diff
@@ -220,14 +220,14 @@ await fileIds.Select(fileId => CheckFileExists(releaseId, fileId, DeletableFileT
             List<ReleaseFile> releaseFiles,
             Release release,
             Stream outputStream,
-            CancellationToken? cancellationToken)
+            CancellationToken cancellationToken)
         {
             using var archive = new ZipArchive(outputStream, ZipArchiveMode.Create);
 
             foreach (var releaseFile in releaseFiles)
             {
                 // Stop immediately if we receive a cancellation request
-                if (cancellationToken?.IsCancellationRequested == true)
+                if (cancellationToken.IsCancellationRequested)
```

Comment:
Not as part of this PR, but should we be adding parameters to BlobStorageService methods to take optional CancellationTokens as well in the future so we can pass them along?  Presumably the underlying SDK libs that we use will have the ability to accept CancellationTokens too.

Reply:
Yes I think it's a good idea to add these in whenever we touch these methods. The storage and other libs of the SDK that we use supports CancellationTokens where appropriate.